forem/forem
# Notifications Initializer Migration to Pack tag

## What type of PR is this? (check all applicable)


## Description

## Related Tickets & Documents



- Related Issue #
- Closes #18998 

## QA Instructions, Screenshots, Recordings

1. Log in with a user that should have notifications(Notifications may have to be created manually)
2. Upon logging in, the notification bell should STILL show the `notification_count` on the notification bell in the upper right corner of the nav.
3. Proceed to the `notifications` page
4. Try saving and liking a post.
5. Also try loading more posts at the bottom if possible.

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/gw3zMXiPDkOyVBsc/giphy.gif)
